python/r2pipe/__init__.py

```diff
@@ -38,7 +38,7 @@
 	r2lang = None
 
 try:
-	import native
+	from .native import RCore
```

Comment:
yeah, it's compatible I think. (see 6.4.2 [modules](https://docs.python.org/2/tutorial/modules.html))